pkg/minibroker/provider.go

```diff
@@ -148,6 +148,14 @@ func NewProvisionParams(m map[string]interface{}) *ProvisionParams {
 	return &ProvisionParams{Object: m}
 }
 
-func buildHostFromService(service corev1.Service) string {
-	return fmt.Sprintf("%s.%s.svc", service.Name, service.Namespace)
+// hostBuilder provides the method for building the OSBAPI service's URI host
+// from a k8s service.
```

Comment:
That's a bit jargon-heavy (I think I'm just having a reaction to `OSBAPI` being so close to `URI` and `k8s` and having to mentally expand them all).

Reply:
Open Service Broker API is no close to URI in any sense. Doesn't make sense at all to compare them.
I don't understand where the jargon is.